gsadjust/pyqt_models.py

```diff
@@ -1224,6 +1261,16 @@ def populate_delta_model(self, loop=None, clear=True):
                                      "GSadjust error")
         return True
 
+
+def survey_serializer(obj, cls, **kwargs):
```

Comment:
This same custom serializer could actually be used for all types, assuming they have .to_json handlers.